src/Compilers/CSharp/Portable/BoundTree/BoundMethodGroup.cs

```diff
@@ -54,6 +54,9 @@ public SyntaxNode NameSyntax
             }
         }
 
+        public SeparatedSyntaxList<TypeSyntax> TypeArgumentsSyntax
+            => NameSyntax is GenericNameSyntax genericName ? genericName.TypeArgumentList.Arguments : default;
```

Comment:
>NameSyntax is GenericNameSyntax genericName ? genericName.TypeArgumentList.Arguments : default; [](http://example.com/codeflow?start=15&length=95)

This looks fragile. If we need this syntax, we probably  should add it to the ```BoundMethodGroup``` as a real property set at construction.